clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java

```diff
@@ -390,6 +390,9 @@ public enum Command {
       "list-store-push-info", "List information about current pushes and push history for a specific store.",
       new Arg[] { URL, CLUSTER, STORE }, new Arg[] { PARTITION_DETAIL_ENABLED }
   ),
+  GET_KAFKA_TOPIC_CONFIGS(
```

Comment:
Nit should we rename it as GET_PUBSUB_TOPIC_CONFIGS to replace kafka with pubsub? Not relate to your change, there are many cases inside using Kafka now. No need to change all of them here.

Reply:
Let's do a one time replacement in a different PR; I hope a global text replacement (KAFKA -> PUBSUB) will do the trick :D